.github/workflows/python-style-check.yml

```diff
@@ -91,6 +91,7 @@ jobs:
         run: bash python/serving/dev/lint-python
 
   create-workflow-badge:
+    if: github.event.pull_request
```

Comment:
does this mean this job will be executed if it is only triggered by PR?

Reply:
Sorry, it should be !github.event.pull_request.